ipams/remote/remote.go

```diff
@@ -30,7 +30,15 @@ func newAllocator(name string, client *plugins.Client) ipamapi.Ipam {
 
 // Init registers a remote ipam when its plugin is activated
 func Init(cb ipamapi.Callback, l, g interface{}) error {
-	plugins.Handle(ipamapi.PluginEndpointType, func(name string, client *plugins.Client) {
+	var handleFunc func(string, func(string, *plugins.Client))
```

Comment:
Same here, you should be able to reduce the footprint of this change, make it less verbose.